Jellyfin.Server/Migrations/MigrationRunner.cs

```diff
@@ -81,5 +85,65 @@ public static void Run(CoreAppHost host, ILoggerFactory loggerFactory)
                 logger.LogDebug("Migration '{Name}' marked as applied in configuration.", migrationRoutine.Name);
             }
         }
+
+        /// <summary>
+        /// Performs a network setting migration.
+        /// </summary>
+        /// <param name="appPaths">The <see cref="IServerApplicationPaths"/>.</param>
+        /// <param name="logger">The <see cref="ILogger"/>.</param>
+        public static void RunSettingsMigration(IServerApplicationPaths appPaths, ILogger logger)
+        {
+            var destFile = Path.Combine(appPaths.ConfigurationDirectoryPath, "network.xml");
+            if (!File.Exists(destFile))
+            {
+                // Migrate version 7.0.3 network configuration.
+                RunSettingMigration<Legacy.V703.ServerConfiguration, Legacy.V703.NetworkConfiguration>(logger, appPaths.SystemConfigurationFilePath, destFile);
+                logger.LogDebug("Migrated to network configuration 7.0.3");
```

Comment:
What does `7.0.3` refer to?

Reply:
10.7.0.3

Reply:
The proper version is `10.7.3`

Reply:
We were at 10.7.0rc3 at the time of submisson.

Hard coded the version as we'd know when the snapshot was taken - but your comment raises an issue - do we keep upping the version number on each release until something changes?

Reply:
I would say set the version to the major.minor version (10.7), as we shouldn't be changing settings in patch versions